core/tracker/inclusion.go

```diff
@@ -423,27 +425,33 @@ func checkAttestationInclusion(sub submission, block block) (bool, error) {
 
 // checkAttestationV2Inclusion checks whether the attestation is included in the block.
 func checkAttestationV2Inclusion(sub submission, block blockV2) (bool, error) {
+	subData, ok := sub.Data.(core.VersionedAttestation)
+	if !ok {
```

Comment:
let's check `subData` is not `nil` just in case

Reply:
You mean if it has filled at least one version in the structure? subData itself cannot be nil.